xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -386,7 +386,7 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
     m_pFormatContext->pb = m_ioContext;
 
-    AVDictionary *options = NULL;
+    AVDictionary *options = GetFFMpegOptionsFromInput();
```

Comment:
Hu, I need more explanations.
You are speaking about DVDInputStreamFFmpeg ?
I have made another try, with a working m3u, and again it's managed by CDVDDemuxFFmpeg (from debug output).